exercices/part3/practice1.py

```diff
@@ -0,0 +1,12 @@
+
+
+if __name__ == "__main__":
+    """Write a program that counts for the user. Let the user enter the starting number, the ending number, 
+    and the amount by which to count."""
+
+    start = int(input("Starting number:  "))
+    end = int(input("Ending number: "))
+    count = int(input("Amount by which to count: "))
```

Comment:
What happens if count ==0? Should be checked that count > 0

Reply:
if the count ==0 an error will display (ValueError: range() arg 3 must not be zero), i will add the check, thank you for the hint, and i apologise for so many lose ends :)